confs2psi.py

```diff
@@ -79,14 +79,14 @@ def confs2psi(insdf, method, basis, spe=False, memory=None):
 
     """
     wdir = os.getcwd()
-    
```

Comment:
Just as a for the record, its good practice to have your file editor automatically remove white space at the end of files. Here's the line for a vimrc if you want it
```
autocmd FileType c,cpp,java,php,python autocmd BufWritePre <buffer> %s/\s\+$//e
```

Reply:
Aghhh I have that in my vimrc but looks like it got a line break in there somehow. Should be fixed now, thanks.